userspace/libsinsp/sinsp.cpp

```diff
@@ -246,6 +270,7 @@ void sinsp::init()
 	if(is_capture())
 	{
 		consume_initialstate_events();
+		import_thread_table();
```

Comment:
note for the reviewer (cc @gnosek ) maybe the refactor wants us to do this import differently?

Reply:
In an ideal world, we'd keep using on_new_entry_from_proc so that it's consistent across all engines. Please remind me, is the issue that we're getting a NULL tinfo when adding a fd from a scap file?

In that case, it would be best to find the tinfo in the savefile engine but that inverts all the layers of abstraction ;) so I guess we can add that null check to on_new_entry_from_proc (if (tinfo==nullptr) { tinfo = find_the_tinfo_in_proc_table(tid) })